bio_page.html

```diff
@@ -0,0 +1,99 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+    <script src="https://code.jquery.com/jquery-3.5.1.min.js"></script>
+    <link href="https://stackpath.bootstrapcdn.com/bootstrap/4.4.1/css/bootstrap.min.css" rel="stylesheet">
+    <script src="https://stackpath.bootstrapcdn.com/bootstrap/4.4.1/js/bootstrap.min.js"></script>
+    <script src="https://stackpath.bootstrapcdn.com/bootstrap/4.4.1/js/bootstrap.bundle.min.js"></script>
+    <link rel="stylesheet" type="text/css" href="style.css">
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Bio Page</title>
+</head>
+<script>
+    window.addEventListener("load", function () { alert("Hello!! Welcome to my Bio page!!") })
+</script>
+<nav class="navbar navbar-expand-lg navbar-dark" style="background-color:rgb(152, 152, 152)">
```

Comment:
was having two nav options intentional? the navbar at the top of the page feels more intuitive from a user perspective, with helpful names of each page to go to